test/realm_test.dart

```diff
@@ -28,8 +28,8 @@ import 'package:timezone/data/latest.dart' as tz;
 import 'package:path/path.dart' as p;
 import 'package:cancellation_token/cancellation_token.dart';
 import '../lib/realm.dart';
-import '../lib/src/realm_class.dart' as realmInternal;
 import 'test.dart';
+import '../lib/src/native/realm_core.dart';
```

Comment:
I prefer `import '../lib/src/realm_class.dart' show RealmInternal;` instead of everything from realm_core.dart.

Reply:
We are not doing that anywhere. The fact that we need to import realm_class explicitly shows we should just import realm_core. that is the reason I have changed this and removed the internal implementation

Reply:
An internal implementation is needed only if called from multiple places from within the SDK libraries.

Reply:
To clarify in other places in the tests we are importing realm_core.dart when we want to work with core internals for some tests. Hence we do it here as well